src/app/environments/environment.ts

```diff
@@ -1,6 +1,7 @@
 import { EnvironmentInterface } from './environment.interface';
 
 export const environment: EnvironmentInterface = {
+  production: false, // during local development (localhost 4000/4200), the user "local@development.com" is authenticated by default
```

Comment:
könnte ich das dann hier auch ausstellen?

Reply:
meinst du allgemein den automatischen Login? wenn du `production: true` setzt in environment.ts (ist eigentlich für dev + stage) dann solltest du das normale Verhalten haben.

Reply:
Maxim und Ole hatten damals zwei files für env variables erstellt: `environment.ts` und `environment.prod.ts`. Ersteres fässt hier jetzt einfach alle Umgebungen zusammen, die nicht Prod sind.

Reply:
ich fügs einfach noch als comment ein :)